### PR TITLE
Subdomain View and integration of jslib

### PR DESCRIPTION
**Pull request by jessgusclark**
Creates subdomain view in the newAdmin. This section's operations.js uses the new js library.

### CODE REVIEW

**Comment by jessgusclark** [comment]
> * When I insert wrong checksum and cancel editting, the checksum message does not disappear.

Good catch! I have fixed this bug and did a force push after rebasing!

File: src/app/tabs/newAdmin/subdomains/components/SubdomainListComponent.js [hunk 1]
```diff
@@ -26,7 +27,9 @@ const SubdomainListComponent = ({
             <SubdomainViewContainer
               key={subdomain.name}
               label={subdomain.name}
+              labelDisplay={`${subdomain.name}.${domain}`}
               value={subdomain.owner}
+              valueDisplay={`${strings.Owner}: ${toChecksumAddress(subdomain.owner, chainId)}`}
```
Comment (jessgusclark) [inline src/app/tabs/newAdmin/subdomains/components/SubdomainListComponent.js]: Why not use `value_prefix` to set strings.owner?

File: src/app/components/AddressInputComponent.js [hunk 2]
```diff
@@ -94,12 +94,12 @@ const AddressInputComponent = ({
     <div className="row addressInput">
       <div className="row view">
         <div className="col-md-3 label">
-          {label}
+          {labelDisplay || label}
```
Comment (jessgusclark) [inline src/app/components/AddressInputComponent.js]: What is the difference between these?

File: src/languages/en.json [hunk 3]
```diff
@@ -278,5 +278,8 @@
   "submit": "Submit",
   "subdomain_owner_set": "Subdomain owner has been set.",
   "delete": "delete",
-  "remove_subdomain_comfirm": "Are you sure you want to remove this subdomain?"
+  "remove_subdomain_comfirm": "Are you sure you want to remove this subdomain?",
+  "purpose_subdomains": "You can create free subdomains for different purposes (personal wallet, gaming, betting...)",
+  "Name": "Name",
+  "Owner": "Owner"
```
Comment (jessgusclark) [inline src/languages/en.json]: these are both duplicate entries. Instead, capitalize the first letter with CSS.

File: src/app/components/AddressInputComponent.js (hunk 2 above)
Comment (ilanolkies) [reply src/app/components/AddressInputComponent.js]: `SubdomainViewContainer` is using `label` to set the new owner.

```
  handleSubmit: newAddress => dispatchProps.handleSubmit(
    stateProps.domain, ownProps.label, newAddress, ownProps.value,
  ),
```

We should display the whole domain

Comment (jessgusclark) [reply src/app/components/AddressInputComponent.js]: ahhh, gotcha. that makes sense.

**Pull request merged**